spk/borgbackup/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = borgbackup
 SPK_VERS = 1.2.0
```

Comment:
```suggestion
SPK_VERS = 1.2.1
```

Reply:
Whoops, sorry. Should have spotted that. Done.

Reply:
If we increase SPK_VER, should SPK_REV be set back to zero?

Reply:
> If we increase SPK_VER, should SPK_REV be set back to one?

No, SPK_REV must never be decreased. It must be incresed to publish new versions to the syno repo.

Reply:
Understood, thanks for the clarification.

Reply:
@adhawkins you find all about the Makefile variables in our wiki at https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables